### ISSUE TITLE
Support for `--locked`?

### ISSUE THREAD

**Issue by kdy1** [baptiste0928/cargo-install#2]
Thank you for the action, I'm using it in swc proejct (https://swc.rs)

Recently `crossbeam-epoch@0.9.8` is published and CI fails with

```

  error[E0658]: trait bounds other than `Sized` on const fn parameters are unstable
     --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/crossbeam-epoch-0.9.8/src/atomic.rs:314:6
      |
  314 | impl<T: ?Sized + Pointable> Atomic<T> {
      |      ^
  ...
  346 |     pub const fn null() -> Atomic<T> {
      |     -------------------------------- function declared as const here
      |
      = note: see issue #93706 <https://github.com/rust-lang/rust/issues/93706> for more information
      = help: add `#![feature(const_fn_trait_bound)]` to the crate attributes to enable
```

so I think the solution is to add `--locked` to cargo install.

**Comment by baptiste0928** [comment]
Thanks for the suggestion! I just released a new version (v1.2.0) with support for a `locked` input that add the `--locked` argument to the install command.

You can use it like this: 
```yaml
- uses: baptiste0928/cargo-install@v1
  with:
    crate: crate-name
    locked: true
```

Thanks for using this action in swc :smile:

**Comment by kdy1** [comment]
Thank you!
